api/envoy/config/route/v3/route_components.proto

```diff
@@ -1712,6 +1712,27 @@ message RateLimit {
       Source source = 4 [(validate.rules).enum = {defined_only: true}];
     }
 
+    // The following descriptor entry is appended with a computed value.
+    // See :ref:`attributes <arch_overview_attributes>` for the set of
+    // available attributes.
+    //
+    // .. code-block:: cpp
+    //
+    //   ("<descriptor_key>", "<expression_value>")
+    message Expression {
```

Comment:
Please add a `[#not-implemented-hide:]` comment here.

Reply:
Implemented.